sdk/communication/communication-call-automation/src/callConnection.ts

```diff
@@ -3,185 +3,235 @@
-        const result = await this.callConnectionImpl.getCall(this.callConnectionId, options);
-        const callConnectionPropertiesDto: CallConnectionPropertiesDto = {
-            ...result,
-            source: result.source ? callSourceDtoConverter(result.source) : undefined,
-            targets: result.targets?.map(target => communicationIdentifierConverter(target))
-        }
-        return callConnectionPropertiesDto;
-    }
+  /**
+   * Get call connection properties
+   */
+  public async getCallConnectionProperties(
+    options: GetCallConnectionPropertiesOptions = {}
+  ): Promise<CallConnectionProperties> {
+    const result = await this.callConnectionImpl.getCall(
+      this.callConnectionId,
+      options
+    );
+    const callConnectionProperties: CallConnectionProperties = {
+      ...result,
+      sourceIdentity: result.sourceIdentity
+        ? communicationIdentifierConverter(result.sourceIdentity)
+        : undefined,
+      targets: result.targets?.map((target) =>
+        communicationIdentifierConverter(target)
+      ),
+      sourceCallerIdNumber: result.sourceCallerIdNumber
+        ? phoneNumberIdentifierConverter(result.sourceCallerIdNumber)
+        : undefined,
+    };
+    return callConnectionProperties;
+  }
 
-    /**
-    * Hang up the call for itself or terminate the whole call.
-    */
-    public async hangUp(
-        isForEveryOne: boolean,
-        options: HangUpOptions = {}
-    ): Promise<void> {
-        if (isForEveryOne) {
-            await this.callConnectionImpl.terminateCall(this.callConnectionId, options);
-        }
-        else {
-            await this.callConnectionImpl.hangupCall(this.callConnectionId, options);
-        }
-        return;
+  /**
+   * Hang up the call for itself or terminate the whole call.
+   */
+  public async hangUp(
+    isForEveryOne: boolean,
+    options: HangUpOptions = {}
+  ): Promise<void> {
+    if (isForEveryOne) {
+      await this.callConnectionImpl.terminateCall(
+        this.callConnectionId,
+        options
+      );
+    } else {
+      await this.callConnectionImpl.hangupCall(this.callConnectionId, options);
     }
+    return;
+  }
 
-    /**
-    * Get a participant from the call
-    */
-    public async getParticipant(
-        participantMri: string,
-        options: GetParticipantOptions = {}
-    ): Promise<CallParticipant> {
-        const result = await this.callConnectionImpl.getParticipant(
-            this.callConnectionId,
-            participantMri,
-            options
-        )
-        const callParticipant: CallParticipant = {
-            identifier: result.identifier ? communicationIdentifierConverter(result.identifier) : undefined,
-            isMuted: result.isMuted
-            }
-        return callParticipant;
-    }
+  /**
+   * Get a participant from the call
+   */
+  public async getParticipant(
+    participantMri: string,
+    options: GetParticipantOptions = {}
+  ): Promise<CallParticipantDto> {
```

Comment:
question: shouldn't this naming the other way around?
CallParticipantDto for models of Rest (the one we send to api service)
CallParticipant for models for devs to consume

Reply:
It is because the internal one is named as CallParticipant and I did not know how to change its name, therefore I used the dto name for the public one. Now I have updated the swagger README.MD file to change the internal one's name. I will change the public one's name later